examples/image-vis-type-example.ipynb

```diff
@@ -0,0 +1,539 @@
7770zSSj5SaaSuwboRaeCed7DI66bU4nxCop8n9EJ4K3exlBOFhlJxE670k8c2E6hBJdCvWjbG49PNlV11OpHfQyy5HySlq0S0k4b3kdiNaTmPcGM43SfTDdLLLCMQMqTkTJWfQcrYUgFikZ84kkofJXWP5/ZcixfgaNVVw4WtqgdwlVPVRcjdERT/GUHTX83a/T9lJFKO/RTyBLI+2386+WXz+zK+foMcG1El9j64LaaiCc+ZKSWbn1bLiiPHVMlscInGEcoHcD2B9lV39YYZ1j9fd9Ld89W//iWbyEbr5i7G+Ccgi7hftcqyzixAZVXUUT0jq1pS5gC59ewb18A6MW0EleKrcoZMyEZaR2bUEcgIIkCIru9G2k5qzUWKOQB0gs6cTB1/DuDUA1ONPcd/sf2HONJjON9M2Z/PU3K8hzWtYOfh2JmY+A0i8a+F8g0DtA5+g1VGcbxGHhTRToA6Q2620s5ei5TShGl+u7XfMi6gEj6DVNEvZ1UVlwK0CH5dZf4fzDUAiyejbIZzXKGHxPmEwug8pNAJLPfoS8+krEAT8dXoFM8x+0/t4yp67nTQ7ekUMf19/vvWm1P3OSpDjEp5Cmrltx7A+xrgqiZqm6wbJ3BCyjB/PqhzCuGE+8dp/AGCh95PFFBkZ9fCeopxmN+J8XCKhC10yKfokal/RyEKEEBkL6WupBPeQ2bVF7Vo/gZJz2GfsiFpOIMjLhQSsa5XH8wpCGJb61yNFj2r0RYxdw7bafawNPV0/zYw7yEWD7+NwVuOeg45AjtNIbixQyqKDEBmBOkig9+Jci3r1I+R2Lb3sqmIaLrwD7ysYN1ycEPKLGK2+Hym6GLeCVvQpYr0TLRZQco7UbCqTkkX/g/FVlChCo76rYYHp9AoCOV2IZLohMlejZ1dxjIPf13v+w2QnjaOnfvHbX8S3TsQ9254Z8xu67E0dbdukpsdJbYtQdtEyxXuB9RohHHNmJYPBYyjRLcQPfEIj/jjHs3MZ7/4kUqRUw68WPd/C0jenEcnDpLaFFP3iSO9qX4dWlLJGiX4KJToosVSUreJ/LObR1USBalKHEFh6ZjuiJLUG6mAxnFLG7+3sqnLIBkK9C3xI37yISng71oeMqQ2cW5lByeMs+XHWJv+KFF2C4AkCXcySh+G91Co3kuXbGR3+T/TT7dTiT1OLP17Oty+g5DRaTaBEIWiZmbNIojvKhN7lBSOfYlFDFBnzE8Mt1sck6igCSyzbWA8egcDgfEjXnENVH2Uovvk538dT9u3tpDi6F4os73xO1367Y/sJO8GbO3H9ieN817cRfi2JWkSJHo6QzEckahHnQ0air2LcCLEax/ghnI9YP/YjuP4V4IfwPqCTn0uoZoqjuV1BNXiMUBU19+HG79HPr2T7touYmvoZvI9wVEplFQjVETK3DoEAAnK7HiFMWR4LkSJFy+kyFzCFEP3yWAyZW0uoDgGCzGzF+iFi/SjGrqUaPMbK2p9TDW9n3/zfsa3xYTxVkvBLLHV/Ee+rRNE9aH2Q+aX3IuUMzkY4N0qgd+LcEM6NYf0wgX6a3GxBq4lCT15NY+xmpOgR6Z0o0S4pNV2sGyiSjmgcUamsOgzI5UGXRPYRQqBEj0pQiFNI2eWW9J9OldaeJzspHD1Bcbb41ed07XN19BOOfeJfod+mcOT0fZ+V4WFSN0bX1stkUQWPxFPn9DXrODL/34nkMXI/yNTibzCTn0VVTdOxK1DCEshZcjeClh08CZHejRSGxf5PkLthDk38foFh8vWi4YVCkTR3K8uTwjAehZYLWDcAonD0QgJJAwLnh1CiTWa3glAYNww+AQQIit5yX0HJhaL1VTi62WsIZB9JykTveow5n2PpZbTCx+hlV4OrodUB+tllpPkl1JKPsNT9ZRb6P0kl/DKBGmcp/VEqwdfQej+d7Fq8T+ibbctxe6EVV1Q2pOwQqknyEuxRvDSPll2Mq5L7kEAYjK+WcsyWntnI57u38Ri3fBdPyyn7RvaCP7o3Cfkx8fAP9Hd2/bESxTxLVc2VrbGLOCTGVdlzdCdKdIsSGykXnDFETS3RtisLfrnoktmxovQmshLJtI0oeIBq8BUq+jEa4ddQooAxZG41gTqKEIZI7ytUXtUueuY0BN2yqw0itW8Z1yRLiedOdjWZXUsS3EWin0KrKarRF/E+LnjtVGjW34dxwyg5R6wfxHuNVhNEosdA/BGGgmIuvRLeRhJ9odjdgx1ATqd/PVLOo+U0UXg/mTmbWnQzqTmTXnZ1yXvvkQSPAEXra25XoOTScs3fl0KNmWsRqQMFMNOOEchFqrJD2xZTdlX9FJI+Q8kn2MeHf5C3/D+8Pbc09r+jRYSENP4N2vn7aZaMWAzh/RSSDIElkItIsmLiyqwl9iGJ2seS2Q5Idu29g9RVUJzo7OrjCFGijcBQC+8mNWfSTl9DrHcWiqhuGCVmQIGnQm5XocqhFiXnMG4MLdr0zHaU6BDIKXI3Ss+cRiv6XDmKmhfyypUPkuYXlHpwxaLgKVpg14y9gbn5n2fF0E8RRHB44gYa8afIzBYG41sJ9C5Gav8Pod5Bc/B+Do5/DMjxVKiEt5Hm55HarWg5x3xJnSmIt2MYN0wtug1j15XH9OHy+L4f7zXWDeDRdM3mIqchunTyswu+nEzp2yFyHyDE19//3A8R+go7eeIHcr9/WOwFv6Mfp/19d/JnJvBU+f/dvY0gLLE6QiCnyX0T7xVL1tE2a5lML1/ehUJ1gLH481T0BKGcJ3MtVNkQY1yrHFaJqYZ34n1AEt5FqPYQqILPJsUCSXgv3sfkdgVJcB/V8EsE6uhyt5xxAyixRCv6HIF+mn5+XsmOW0cnfS2ZPQMtjxGovfTzS5H0qURfYm7+rUDO/OJ/Iksh0jvIzBa0nCYJ7qKXXY33IZk5i/2HPs1A/f+kKN8t0MsuYyF9FcYN4XyCoEuon8T5ZhmK9FFi/t904DmfFDpsgJTtcvCng/UVrK8QyMWyiSdAy
```

Comment:
@vincentsarago let's change this to an async URL load from a local file, if possible, to reduce file size of the notebook and encourage async loading of images.

Reply:
Oh well I didn't knew mapboxgl could read a local file 👌